### ISSUE TITLE
Error: Cannot find module '@commander-js/extra-typings'

### ISSUE THREAD

**Issue by MonoLogueChi** [openupm/openupm-cli#87]
When I used openupm, I encountered an error as follows

My environment is `node 20.10.0 + pnpm 8.13.1 + openupm-cli 1.19.3` , `node 20.10.0 + npm 10.2.3 + openupm-cli 1.19.3` has the same error

```
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '@commander-js/extra-typings'
Require stack:
- C:\Users\mc\AppData\Local\pnpm\global\5\.pnpm\openupm-cli@1.19.3\node_modules\openupm-cli\lib\cli.js
- C:\Users\mc\AppData\Local\pnpm\global\5\.pnpm\openupm-cli@1.19.3\node_modules\openupm-cli\lib\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (C:\Users\mc\AppData\Local\pnpm\global\5\.pnpm\openupm-cli@1.19.3\node_modules\openupm-cli\lib\cli.js:6:25)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\mc\\AppData\\Local\\pnpm\\global\\5\\.pnpm\\openupm-cli@1.19.3\\node_modules\\openupm-cli\\lib\\cli.js',
    'C:\\Users\\mc\\AppData\\Local\\pnpm\\global\\5\\.pnpm\\openupm-cli@1.19.3\\node_modules\\openupm-cli\\lib\\index.js'
  ]
}

Node.js v20.10.0
```

**Comment by ComradeVanti** [comment]
Just confirmed this error on my Linux machine. I'm looking into it now.

**Comment by ComradeVanti** [comment]
The issue was that `@commander-js/extra-typings` was declared as a dev-dependency. This is fixed in https://github.com/openupm/openupm-cli/pull/91.

**Comment by favoyang** [comment]
Closed by #91 (v1.19.4). Thanks to @MonoLogueChi for reporting and to @ComradeVanti for fixing it.

**Comment by favoyang** [comment]
@all-contributors please add @MonoLogueChi for bug.
